src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/settings/dto/NewProjectMemberRequest.java

```diff
@@ -0,0 +1,25 @@
+package ca.corefacility.bioinformatics.irida.ria.web.projects.settings.dto;
+
```

Comment:
Add a comment to indicate what this is used for

Reply:
Done